Bluejay.asm

```diff
@@ -4238,6 +4366,7 @@ exit_run_mode_on_timeout:
 
 exit_run_mode:
 	clr	IE_EA					; Disable all interrupts
+	clr Flag_Ext_Tele			; Clear extended DSHOT telemetry flag
```

Comment:
I don't care much about what formatting we use as long as it is consistent. Some places it is 1 space, some places it is 4 spaces. Some places tabs. So i say we use  some git commit hooks to run a formatter before commit.

Note, this is NOT a suggestion for this pull request. Has to be done in a separate pull request - when done.

Reply:
Yes, totally, I already added this to the [cleanup task](#13).